### PR TITLE
#6-subscibe-footer-1

### PR DESCRIPTION
**Pull request by Dawindows**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -189,6 +189,124 @@ <h1>New Providence is the great Ui ki</h1>
...
+                <ul class="footer-1">
+                    <li class="footer-2">SUPPORT</li>
+                    <li class="footer-3"><a href="">F.A.Q.</a></li>
+                    <li class="footer-3"><a href="">Contact us</a></li>
+                    <li class="footer-3"><a href="">Live chat</a></li>
+                    <li class="footer-3"><a href="">Phone call</a></li>
+                </ul>
+            </div>
+        </div>
+        </div>
```
Comment (BobrovSergey) [inline index.html]: This `</div>` tag is unnecessary and should be deleted

File: index.html [hunk 2]
```diff
@@ -189,6 +189,124 @@ <h1>New Providence is the great Ui ki</h1>
...
+    <div class="footer">
+        <div class="footer-content">
+            <div class="footer-left">
+                <div class="logo-1"><img src="assets/img/Logo.png" alt=""></div>
+                <p class="txt-logo-1">it was some time befor he obtained any
+                    <br> answer, and replay, when made, was
+                    <br> unpropitious.</p>
+                <div class="button-7"><button class="footer-button">
+                    <i class="fab fa-apple"></i>
+                    <div class="footer-button-txt">Download on the <p>App Store</p></div></i></button></div>
```
Comment (BobrovSergey) [inline index.html]: There is excess `</i> tag, should be removed

File: index.html [hunk 3]
```diff
@@ -189,6 +189,124 @@ <h1>New Providence is the great Ui ki</h1>
...
+                    <li><i class="fab fa-pinterest-p"></i></li>
+                    <li><i class="fab fa-linkedin-in"></i></li>
+                </ul>
+            </div>
+            <div class="footer-right">
+                <ul class="footer-1">
+                    <li class="footer-2">LEARN MORE</li>
+                    <li class="footer-3"><a href="">How it works?</a></li>
+                    <li class="footer-3"><a href="">Meeting tool</a></li>
+                    <li class="footer-3"><a href="">Live striming</a></li>
```
Comment (BobrovSergey) [inline index.html]: Typo in the `striming` word

File: index.html [hunk 4]
```diff
@@ -189,6 +189,124 @@ <h1>New Providence is the great Ui ki</h1>
...
+                    <li class="footer-3"><a href="">Contat method</a></li>
```
Comment (BobrovSergey) [inline index.html]: Typo in the `Contat` word

**Comment by BobrovSergey** [comment]
It is a common practice to add Description in your PR with short list of what was done
And pay attention to the commit messages - they should be informative and contain brief description of change
Overall looks great 👍

File: index.html (hunk 1 above)
Comment (Dawindows) [reply index.html]: > Этот `</div>`тег не нужен и должен быть удален

deleted

File: index.html (hunk 2 above)
Comment (Dawindows) [reply index.html]: > There is excess `</i>` tag, should be removed

deleted

File: index.html (hunk 3 above)
Comment (Dawindows) [reply index.html]: > Typo in the `striming` word

fixed

File: index.html (hunk 4 above)
Comment (Dawindows) [reply index.html]: > Typo in the `Contat` word

fixed

**Pull request merged**